spec/models/guest_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe Guest do
+  describe "relationships" do
+    it { should have_many :guest_rooms }
+    it { should have_many(:rooms).through(:guest_rooms) }
```

Comment:
Per your comment on this PR: Using this assertion with `through` is more precise of a test. Tables can be associated through different routes (think of a db diagram where there's a circle of relationships so there could be relationships set up from either direction). Typically there's a more direct association that we set up in the model, so we want to test for that precise association path rather than just testing that it's associated.